src/bit-systems/waypoint.ts

```diff
@@ -164,6 +171,16 @@ export function waypointSystem(
       }
     }
 
+    const hashUpdated = window.location.hash !== "" && previousWaypointHash !== window.location.hash;
+    if (hashUpdated && initialSpawnHappened) {
+      const waypointName = window.location.hash.replace("#", "");
+      if (eid) {
```

Comment:
What is this `eid` check for?

Reply:
Yeah, I think I added it at some point and I forgot to remove. Thanks.